src/software/ros/basics/packages.md

```diff
@@ -0,0 +1,23 @@
+# ROS Packages
+ROS software is organized into packages, each of which contains some combination of code, data, and documentation. Usually there is a package for each 'function' of your robot. For example, you can have a package for navigation or for sensor capturing.
+Thanks to the contributing community, there are many pre built packages available to download for free.
+
+Let's create our first package!
+
+* Creating a packages
+```
+cd ~/catkin_ws/src
+catkin_create_pkg my_awesome_code rospy
```

Comment:
Maybe explain why `rospy` is added at the end of the command? And talk about package dependencies.